backend/src/routes/authRoutes.js

```diff
@@ -0,0 +1,11 @@
+import express from "express";
+import auth from "../middlewares/auth.js";
+
+const router = express.Router();
+
+//auth test
+router.post("/check", auth, async (req, res) => {
```

Comment:
Please implement error handling for this route, similar to what you have done in logRoutes, in case the auth check returns an error

Reply:
I believe if auth.js encounters an error, it already returns a res? i modified the auth.js to also log the error in the console, but because in auth,js if the middleware encounters an error it returns so the .post block doesnt get executed?

Reply:
because the auth check only runs the middleware